vite.config.mts

```diff
@@ -35,6 +35,7 @@ export default defineConfig(({mode}) => {
       extensions: ['.ts', '.js', '.vue'],
     },
     server: {
+      hmr: false,
```

Comment:
hmr was freezing tabs while I was working on route guarding. Hope folks don't mind it being off until we have time to figure that out.